build.sbt

```diff
@@ -535,6 +535,9 @@ lazy val testsBase = circeCrossModule("tests", mima = None)
       "org.typelevel" %%% "discipline-scalatest" % disciplineScalaTestVersion,
       "org.typelevel" %%% "discipline-munit" % disciplineMunitVersion
     ),
+    libraryDependencies ++= {
+      if (isScala3.value) Seq("org.scala-lang" %% "scala3-staging" % scalaVersion.value) else Nil
```

Comment:
Is this only used for tests?  Can it be put into the `Test` config and avoid a transitive dependency?

If not, how stable is that dependency?  It looks like it depends on the full version of Scala?  Would the same circe-core_3 still work on Scala 3 of at least the same minor version as Circe was released with?